lib/chef/provider/powershell_script.rb

```diff
@@ -29,20 +29,6 @@ def initialize (new_resource, run_context)
         add_exit_status_wrapper
       end
 
-      def action_run
-        validate_script_syntax!
-        super
-      end
-
-      def command
```

Comment:
It's already in the resource and windows_script handles dealing with whether or not to use file redirection paths.  it only exists here for the validate step, which I moved into the wrapper script so we only have on shell out and don't have to deal with re-implementing the "should I use file system bitness redirection or not".